src/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -158,7 +159,7 @@ public static void DuplicateKeysFail()
         public static void DictionaryOfObject()
         {
             {
-                IDictionary obj = JsonSerializer.Parse<IDictionary>(@"{""Key1"":1}");
```

Comment:
Why remove the `IDictionary` test case rather than just adding the `Dictionary<string, object>` one?